aspnetcore/security/cors/3.1sample/Cors/WebAPI/wwwroot/js/MyJS.js

```diff
@@ -0,0 +1,21 @@
+﻿function MyTestCors3(host, uri, methodName, includeHeaders=false) {
+    const resultSpan = document.getElementById('result');
+    const myHeaders = includeHeaders ? { 'Content-Type': 'x-custom-header' } : {};
+
+    fetch(host + uri,
```

Comment:
String interpolation is slightly more elegant:

```suggestion
    fetch(`${host}${uri}`,
```